portal/static/js/main.js

```diff
@@ -249,21 +249,46 @@ var CONSENT_ENUM = {
 
 var fillContent = {
     "clinical": function(data) {
+
+        // sort from newest to oldest
+        data.entry.sort(function(a,b){
+            var dateA = (new Date(a.content.meta.lastUpdated)).getTime() / 1000;
+            var dateB = (new Date(b.content.meta.lastUpdated)).getTime() / 1000;
+            if (dateB < dateA) {
+                return -1;
+            } else if (dateB > dateA) {
+                return 1;
+            } else {
+                return 0;
+            };
+        });
         $.each(data.entry, function(i,val){
             var clinicalItem = val.content.code.coding[0].display;
+            var clinicalValue = val.content.valueQuantity.value;
+            var status = val.content.status;
             if (clinicalItem == "PCa diagnosis") {
                 clinicalItem = "pca_diag";
             } else if (clinicalItem == "PCa localized diagnosis") {
                 clinicalItem = "pca_localized";
-            }
-            var ci = $('div[data-topic="'+clinicalItem+'"]');
-            if (ci.length > 0) ci.fadeIn().next().fadeIn();
-            var clinicalValue = val.content.valueQuantity.value;
+            };
             var $radios = $('input:radio[name="'+clinicalItem+'"]');
             if ($radios.length > 0) {
-                if($radios.is(':checked') === false) {
-                    $radios.filter('[value='+clinicalValue+']').prop('checked', true);
-                }
+                if(!$radios.is(':checked')) {
+                    if (status == "unknown") $radios.filter('[value="unknown"]').prop('checked', true);
+                    else $radios.filter('[value='+clinicalValue+']').prop('checked', true);
+                    if (clinicalItem == "biopsy") {
+                        if (clinicalValue == "true") {
+                            var issuedDate = val.content.issued;
+                            var d = issuedDate.substring(0, 10).split("-");
+                            //in DD/MM/YYYY format
+                            $("#biopsyDate").val(d[2] + "/" + d[1] + "/" + d[0]);
```

Comment:
I worry this is a little fragile, does it make sense to use `.toLocaleString()` everywhere, or do you think that's overkill?